spec/services/post_alerter_spec.rb

```diff
@@ -410,12 +410,75 @@ def create_post_with_alerts(args = {})
     end
 
     it "triggers :before_create_notifications_for_users" do
-      post = Fabricate(:post, raw: '[quote="EvilTrout, post:1"]whatup[/quote]')
+      post = Fabricate(:post, raw: '[quote="evilTrout, post:1"]whatup[/quote]')
```

Comment:
```suggestion
      post = Fabricate(:post, raw: '[quote="eviltrout, post:1"]whatup[/quote]')
```

right?

Reply:
We downcase all of the usernames so technically it does not matter, I just changed the other examples for consistencies sake... I'll update this one too, missed it by accident.